Watermark Maker(On Image)/Watermarker(Image).py

```diff
@@ -0,0 +1,110 @@
+#Import required Image library
+from PIL import Image, ImageDraw, ImageFont
+
+#Taking Input From user
+path = input("Input the path of the image: ")
+path = path.strip('""')
+im = Image.open(path)
+width, height = im.size
+
+#Taking input for the text
+text = input('Enter the text for the  watermark:' )
+font = ImageFont.truetype('arial.ttf', 20 )
+
+#Taking Input For the Position of the text
+print("Where you want to input the text . Pls enter\n 1 - Top Left \n 2 - Top Right \n 3 - Bottom Left \n 4 - Bottom Right \n 5 - Center")

```

Comment:
Here, the choice is ambiguous. One is not sure whether to input is the number associated with position or the position itself. Add a print statement to clarify this